schemas/context/destination.example.1.json

```diff
@@ -0,0 +1,10 @@
+{
+  "xdm:destinationID": "1234",
```

Comment:
destinationID is a string, and not numeric. Example of a Destination ID: `b297ed27-607e-4715-97ed-27607e171589`